cmd/webhook/app/server.go

```diff
@@ -108,6 +111,13 @@ func (a *admissionHandler) mutateResource(ar v1beta1.AdmissionReview) *v1beta1.A
 	}
 	annotations := crd.GetAnnotations()
 
+	// Do not change the userInfo if the object is being created by the AppMgr.
+	// This value is stamped onto the Subscription object by Appmgr (Only for subscriptions)
+	if crd.Kind == "subscription" && ar.Request.UserInfo.Username == UserIDforAppMgr {
```

Comment:
Subscription?

@zhiweiyin318 as we discussed, the resource here should not be crd.

Reply:
Is it somewhere else in the code then? As I'm looking to NOT block when the resource type is a "Subscription" kind and the userID is the AppMgr service account.

Reply:
Yeah, i think the kind should be capitalized